easybuild/easyconfigs/f/FigureGen/FigureGen-51-foss-2019a.eb

```
@@ -0,0 +1,40 @@
+easyblock = 'CmdCp'
+
+name = 'FigureGen'
+version = '51'
+
+homepage = 'https://ccht.ccee.ncsu.edu/category/software/figuregen/'
+description = """  FigureGen is a Fortran program that creates images for ADCIRC files.
+It reads mesh files (fort.14, etc.), nodal attributes files (fort.13, etc.) and output 
+files (fort.63, fort.64, maxele.63, etc.). It plots contours, contour lines, and vectors. 
+Using FigureGen, you can go directly from the ADCIRC input and output files to a 
+presentation-quality figure, for one or multiple time snaps.  """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'usempi': True}
+
+github_account = 'ccht-ncsu'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': 'master.zip', 'filename': SOURCE_ZIP}]
+checksums = ['4199816638cf95ccd661eb0509c46d80c9dae986b2053a2db23cce293dcfa328']
```

Use the sources from the commit defined above
```suggestion
sources = ['%s.tar.gz' % local_commit]                                          
checksums = ['b33214ffb339a779364a78ac988f6acb396e5a53fdaaea30ddd02ed8a7f66fd9']
```